modules/pytket-quantinuum/tests/backend_test.py

```diff
@@ -519,3 +530,22 @@ def test_custom_api_handler(device_name: str) -> None:
 
     assert backend_1._api_handler is not backend_2._api_handler
     assert backend_1._api_handler._cred_store is not backend_2._api_handler._cred_store
+
+
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
+@pytest.mark.parametrize(
+    "authenticated_quum_backend", [{"device_name": "H1-1SC"}], indirect=True
+)
+def test_wasm(
+    authenticated_quum_backend: QuantinuumBackend,
+) -> None:
+    wasfile = WasmFileHandler(str(Path(__file__).parent / "sample_wasm.wasm"))
```

Comment:
Have you created a new file or have you used the other one I have uploaded?

Reply:
it's a new file, with some working compiled wasm